src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp

```diff
@@ -193,13 +192,19 @@ namespace {
 std::map<std::string, std::string> emptyAdditionalConfig;
 
 std::vector<std::map<std::string, std::string>> additionalConfig {
+#ifndef OV_CPU_WITH_MLAS
+    // empty config runs in FP32 and mlas is used
     std::map<std::string, std::string>{/* empty config */},
+#endif
     {{PluginConfigParams::KEY_ENFORCE_BF16, PluginConfigParams::YES}}
 };
 
 std::vector<std::map<std::string, std::string>> filterAdditionalConfig_Brgemm() {
```

Comment:
General ask for test changes. Can we avoid modification of current test instances and add additional ones to directly cover mlas?  Current changes assumes we throw out most of the brgemm test cases is case the plugin is built with mlas which results in worse coverage. 
So what we need to do is to exclude `empty` and 'bias' fusing configs from `brgemm` instances since such cases are now mapped to mlas impl and add new instances for them.
@zhangYiIntel  what do you think?

Reply:
Thanks for your proposal, it's more reasonable to separate different tests. I have implement this in latest commit